data/tools/GUI.pyw

```diff
@@ -1004,7 +1049,9 @@ class WmlindentTab(Frame):
         self.mode_frame.grid(row=0,
                              column=0,
                              sticky=N + E + S + W)
+
         self.radio_normal = Radiobutton(self.mode_frame,
+                                        # TRANSLATORS: Normal run mode for wmlindent.
```

Comment:
The presence of separate translator notes suggests that this `_("Normal")` and the other `_("Normal")` are supposed to be separate strings that can receive different translations. How are we going to handle this here? Or, do we just accept they're one string? The normal `tag^` syntax doesn't apply since this is being passed to Tk.

Reply:
I wondered about that. I think they're just meant to be one string - I'll update the notes to refer to the tools generically.